packages/admin/admin/src/form/FinalFormSelect.tsx

```diff
@@ -1,12 +1,14 @@
-import { CircularProgress, InputAdornment, MenuItem, Select, type SelectProps } from "@mui/material";
+import { CircularProgress, InputAdornment, MenuItem, Select, type SelectProps, Typography } from "@mui/material";
 import { type ReactNode } from "react";
 import { type FieldRenderProps } from "react-final-form";
 import { FormattedMessage } from "react-intl";
 
 import { ClearInputAdornment } from "../common/ClearInputAdornment";
 import { type AsyncOptionsProps } from "../hooks/useAsyncOptionsProps";
+import { MenuItemDisabledOverrideOpacity } from "./FinalFormSelect.sc";
 
 export interface FinalFormSelectProps<T> extends FieldRenderProps<T, HTMLInputElement | HTMLTextAreaElement> {
+    getNoOptionsLabel?: () => ReactNode;
```

Comment:
This doesn't have to be a function: `noOptionsLabel?: ReactNode`. What do you think?

Reply:
ofc ReactNode is enough

Reply:
Are we fine with the `get` Prefix?

I mean we could also name it e.g. `noOptionsLabel`

Reply:
I'd remove it (as suggested 😁)

Reply:
updated `getNoOptionsLabel` to `noOptionsLabel`